integration/js/pages/TestAppPage.js

```diff
@@ -53,6 +70,12 @@ class TestAppPage {
     await attendeeNameInputBox.sendKeys(attendeeName);
   }
 
+  async enterMeetingTitle(meetingTitle) {
+    await this.enterMeetingId(meetingTitle);
+    let joinMeetingButton = await this.driver.findElement(elements.joinMeetingButton);
+    await joinMeetingButton.click();
+  }
+
```

Comment:
This looks the same as authenticate(). We can use one and remove the other?

Reply:
But the authenticate() has step to create meetingId randomly. In this end_to_end test, we are relying on sever.js to create attendeeId and meetingID for us, then we type in these values into the UI text input.

Reply:
I noticed that you stil create the meeting id randomly in integration/js/VideoEnd2EndTest.js. The code looks almost identical for AuthenticateUserStep